src/index.ts

```diff
@@ -1,8 +1,4 @@
-// @flow
-
 export {
   default as createLightship,
 } from './factories/createLightship';
-export type {
-  LightshipType,
-} from './types';
+export * from './types';
```

Comment:
I would rather explicitly export only the types that are potentially to be used by the user.

This is because otherwise every type change becomes a breaking change.

Reply:
Makes sence. I added ConfigurationInput. One might want to use that type in application code. The others are all internal as far as I see.